packages/auth-server/pages/dashboard/settings/index.vue

```diff
@@ -103,6 +89,45 @@
             </Button>
           </div>
         </Card>
+
+        <Card
+          v-for="method in activeOidc"
+          :key="method.digest"
+          :loading="getOidcAccountsInProgress"
+          class="p-6"
+        >
+          <div class="flex justify-between lg:flex-row flex-col items-start gap-4">
+            <div class="space-y-4">
+              <div class="flex items-center gap-2">
+                <h3 class="font-semibold text-lg">
+                  Google Recovery
+                </h3>
+              </div>
+              <div
+                class="flex flex-col items-start gap-3 text-gray-600 dark:text-gray-400"
+              >
+                <div class="flex items-center gap-3">
+                  <ShieldCheckIcon class="w-5 h-5 flex-shrink-0" />
+                  <span class="font-mono text-sm">https://accounts.google.com</span>
```

Comment:
Should extract `iss` from contract

```suggestion
                  <span class="font-mono text-sm">{method.iss}</span>
```